dohq_artifactory/admin.py

```diff
@@ -740,6 +742,7 @@ def _create_json(self):
             "propertySets": [],
             "archiveBrowsingEnabled": self.archiveBrowsingEnabled,
             "yumRootDepth": 0,
+            "maxUniqueTags": self.maxUniqueTags,
```

Comment:
is that supported by API V1 ?

Reply:
V1 does not support it https://www.jfrog.com/confluence/display/JFROG/Using+Docker+V1

Reply:
@allburov , thanks for checking
then I would request handling of the property depending on the API version